MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.h

```diff
@@ -206,8 +208,8 @@ vtkMRMLMarkupsToModelNode
   bool CleanMarkups;
   bool ButterflySubdivision;
   double DelaunayAlpha;
+  bool ConvexHull;
```

Comment:
Do you save this in the scene and reload the value from the scene? (there are read/write XML methods in the MRML node cxx)